conduit/pipeline/pipeline_test.go

```diff
@@ -612,6 +612,7 @@ func TestRoundRequestError(t *testing.T) {
 func TestRoundOverride(t *testing.T) {
 	// cli override NextRound, 0 is a test for no override.
 	for i := 0; i < 10; i++ {
+		i := i
```

Comment:
good catch

Reply:
see cbe1692504021f71c69af1b636aa00cbe6015285

FYI - it didn't actually catch the problem. But we're also using it in `go-algorand` so it should be safe.